Client/Frontend/Fakespot/Views/FakespotOptInCardView.swift

```diff
@@ -0,0 +1,359 @@
+        button.titleLabel?.numberOfLines = 0
+        button.addTarget(self, action: #selector(self.didTapSecondaryButton), for: .touchUpInside)
+        button.titleLabel?.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                                         size: UX.secondaryButtonFontSize)
+    }
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: Constraints Setup
+    private func setupLayout() {
+        addSubviews(cardContainer, mainView)
+        mainView.addSubview(contentStackView)
+        contentStackView.addArrangedSubview(headerLabel)
+
+        contentStackView.addArrangedSubview(bodyFirstParagraphLabel)
+        contentStackView.addArrangedSubview(learnMoreButton)
+
+        contentStackView.addArrangedSubview(optInImageView)
+
+        disclaimerStackView.addArrangedSubview(disclaimerTextLabel)
+        disclaimerStackView.addArrangedSubview(privacyPolicyButton)
+        disclaimerStackView.addArrangedSubview(termsOfUseButton)
+        contentStackView.addArrangedSubview(disclaimerStackView)
+
+        contentStackView.addArrangedSubview(mainButton)
+        contentStackView.addArrangedSubview(secondaryButton)
+
+        NSLayoutConstraint.activate([
+            cardContainer.leadingAnchor.constraint(equalTo: leadingAnchor),
+            cardContainer.topAnchor.constraint(equalTo: topAnchor),
+            cardContainer.trailingAnchor.constraint(equalTo: trailingAnchor),
+            cardContainer.bottomAnchor.constraint(equalTo: bottomAnchor),
+
+            contentStackView.topAnchor.constraint(equalTo: mainView.topAnchor,
+                                                  constant: UX.contentStackViewPadding),
+            contentStackView.bottomAnchor.constraint(equalTo: mainView.bottomAnchor,
+                                                     constant: -UX.contentStackViewPadding),
+            contentStackView.leadingAnchor.constraint(equalTo: mainView.leadingAnchor,
+                                                      constant: UX.contentStackViewPadding),
+            contentStackView.trailingAnchor.constraint(equalTo: mainView.trailingAnchor,
+                                                       constant: -UX.contentStackViewPadding),
+        ])
+    }
+
+    // MARK: Button Actions
+    @objc
+    private func didTapLearnMore() {
+        viewModel?.onTapLearnMore()
+    }
+
+    @objc
+    private func didTapTermsOfUse() {
+        viewModel?.onTapTermsOfUse()
+    }
+
+    @objc
+    private func didTapPrivacyPolicy() {
+        viewModel?.onTapPrivacyPolicy()
+    }
+
+    @objc
+    private func didTapMainButton() {
+        viewModel?.shouldOptIn = true
+        viewModel?.onTapMainButton()
+    }
+
+    @objc
+    private func didTapSecondaryButton() {
+        
```

Comment:
this will be a swiftlint warning
```suggestion
    private func didTapSecondaryButton() {
```